peeweelearn.py

```diff
@@ -0,0 +1,75 @@
+from peewee import *
```

Comment:
Is this file being used? Seems to me like it could be deleted

Reply:
Alright